console/console/about_command.go

```diff
@@ -0,0 +1,156 @@
+package console
+
+import (
+	"fmt"
+	"runtime"
+	"sort"
+	"strings"
+
+	"github.com/goravel/framework/contracts/console"
+	"github.com/goravel/framework/contracts/console/command"
+	"github.com/goravel/framework/contracts/foundation"
+	"github.com/goravel/framework/support/str"
+)
+
+type AboutCommand struct {
+	app foundation.Application
+}
+
+type information struct {
+	section map[string]int
+	details [][]kv
+}
+type kv struct {
+	key   string
+	value string
+}
+
+var appInformation = &information{section: make(map[string]int)}
+var customInformationResolvers []func()
+
+func NewAboutCommand(app foundation.Application) *AboutCommand {
+	return &AboutCommand{
+		app: app,
+	}
+}
+
+// Signature The name and signature of the console command.
+func (receiver *AboutCommand) Signature() string {
+	return "about"
+}
+
+// Description The console command description.
+func (receiver *AboutCommand) Description() string {
+	return "Display basic information about your application"
+}
+
+// Extend The console command extend.
+func (receiver *AboutCommand) Extend() command.Extend {
+	return command.Extend{
+		Flags: []command.Flag{
+			&command.StringFlag{
+				Name:  "only",
+				Usage: "The section to display",
+			},
+		},
+	}
+}
+
+// Handle Execute the console command.
+func (receiver *AboutCommand) Handle(ctx console.Context) error {
+	receiver.gatherApplicationInformation()
+	ctx.NewLine()
+	appInformation.Range(ctx.Option("only"), func(section string, details []kv) {
+		ctx.TwoColumnDetail("<fg=green;op=bold>"+section+"</>", "")
+		for i := range details {
+			ctx.TwoColumnDetail(details[i].key, details[i].value)
+		}
+		ctx.NewLine()
+	})
+	return nil
+}
+
+// gatherApplicationInformation Gather information about the application.
+func (receiver *AboutCommand) gatherApplicationInformation() {
+	configFacade := receiver.app.MakeConfig()
+	appInformation.addToSection("Environment",
+		"Application Name", configFacade.GetString("app.name"),
+		"Goravel Version", str.Of(receiver.app.Version()).LTrim("v").String(),
+		"Go Version", str.Of(runtime.Version()).LTrim("go").String(),
+		"Environment", configFacade.GetString("app.env"),
+		"Debug Mode", func() string {
+			if configFacade.GetBool("app.debug") {
+				return "<fg=yellow;op=bold>ENABLED</>"
+			}
+			return "OFF"
+		}(),
+		"URL", str.Of(configFacade.GetString("http.url")).Replace("http://", "").Replace("https://", "").String(),
```

Comment:
Is the `Replace` required? If not, I think it's better to print `http.url` directly.